apps/core/src/components/inputs/SendTokenFormInput.tsx

```diff
@@ -59,6 +57,12 @@ export function SendTokenFormInput({
         ? formattedGasBudgetEstimation + ' ' + gasToken
         : undefined;
 
+    const totalBalance = coins.reduce((acc, { balance }) => {
+        return BigInt(acc) + BigInt(balance);
+    }, BigInt(0));
+    const approximation =
+        parseAmount(values.amount, coinDecimals) === totalBalance &&
+        symbol === IOTA_COIN_METADATA.symbol;
```

Comment:
is this robust enough? I think I can create a coin with the IOTA symbol and then this logic will hit, wouldnt basing it on the coin type be more robust?